src/main/java/seedu/mark/model/bookmark/Remark.java

```diff
@@ -9,14 +9,17 @@
  */
 public class Remark {
 
-    public static final String MESSAGE_CONSTRAINTS = "Remarks can take any values, and it should not be blank";
+    public static final String SPECIAL_CHARACTER = "/";
+
+    public static final String MESSAGE_CONSTRAINTS = "Remarks can contain any characters except " + SPECIAL_CHARACTER;
 
     /*
      * The first character of the remark must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
```

Comment:
You can mention the special character in the javadoc comment as well

Reply:
I left it out as it's written in `MESSAGE_CONSTRAINTS`, directly above this. Should also include it here?

Reply:
Can we rename it to INVALID_CHARACTER instead?

Reply:
Done! Okay to merge?